adserver/admin.py

```diff
@@ -969,7 +973,7 @@ class RegionTopicAdmin(admin.ModelAdmin):
 
 # Don't register Impression Admin's outside dev, since they will just 502 from too much data.
 if settings.DEBUG:
-    admin.site.register(AdImpression, AdImpressionsAdmin)
+    admin.site.register(AdImpression, ImpressionAdmin)
     admin.site.register(UpliftImpression, AdImpressionsAdmin)
```

Comment:
Maybe if we're going to use a different admin from `UpliftImpression`, we should just change the name of that one. eg. AdImpressionsAdmin -> UpliftImpressionAdmin, AdImpression -> AdImpressionAdmin.